src/main/java/com/ghilly/service/CityServiceRest.java

```diff
@@ -49,6 +50,16 @@ public List<City> getAllCities() {
         return cities;
     }
 
+    @Override
+    public void update(int cityId, String newName) {
```

Comment:
we should to get convrsation from Conntroller to Service via class City 
You should do the same  for all methods